packages/devtools_app/pubspec.yaml

```diff
@@ -64,6 +64,7 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   mockito: ^5.0.9
+  vector_math: ^2.0.0
```

Comment:
why is this in dev_dependenicies and not dependencies? Is the inspector_polyfil_script not used in a release build at runtime? @jacob314

Reply:
I believe that this script - `packages/devtools_app/assets/scripts/inspector_polyfill_script.dart` - is not used by the devtools source itself, but is eval'd in the context of the running flutter app. So it really needs `vector_math` to be in the deps of the app under debug. We just add an entry into the devtools pubspec so that we can get complete analysis results for the file - to know that it analyzes cleanly.

Completely aside, we might want to work to reduce the amount of code that we're evaling in the running app - to incrementally push more of the functionality into the in-app inspector implementation.